src/components/about-us/MissionSection.svelte

```diff
@@ -53,7 +53,9 @@
     <div class="mission-card">
       <img src={aboutUsImage.src} alt={aboutUsImage.alt} />
       <h3>{heading}</h3>
-      {@html description.html}
+      <p>
+      {@html description.inlineHtml}
+      </p>
```

Comment:
What is the rationale for this change?

Reply:
Because using `:global(p)` is just not very nice to me. Trying to avoid it where possible.